app/src/main/java/ar/com/wolox/android/example/model/News.kt

```diff
@@ -1,12 +1,22 @@
 package ar.com.wolox.android.example.model
 
+import java.io.Serializable
+
 data class News(
     val id: Int,
-    val commenter: String,
-    val comment: String,
-    val date: String,
-    val avatar: String,
-    val likes: List<Int>,
+    var commenter: String,
+    var comment: String,
+    var date: String,
+    var avatar: String,
+    var likes: MutableList<Int>,
     val createdAt: String,
     val updatedAt: String
-)
\ No newline at end of file
+) : Serializable {
+    fun update(new: NewDetailResponse) {
```

Comment:
Here you are using the word `new` to refer to a single item of news. The word `news` in english is singular and uncountable so you cannot do this. You can read more about it here if you want: https://www.espressoenglish.net/is-news-singular-or-plural/
I would suggest you to replace `new` by `newsItem` or something like that.